pkg/network/ovn_kubernetes.go

```diff
@@ -2240,7 +2240,7 @@ func isInterConnectEnabledOnDaemonset(ds *appsv1.DaemonSet, containerName string
 }
 
 func isInterConnectEnabledOnNodeDaemonset(ds *appsv1.DaemonSet) bool {
-	return isInterConnectEnabledOnDaemonset(ds, util.OVN_NODE)
+	return isInterConnectEnabledOnDaemonset(ds, util.OVN_NODE) || isInterConnectEnabledOnDaemonset(ds, util.OVN_CONTROLLER)
```

Comment:
hmm is this for backwards compatibility we check this for upgrades of sorts? since we don't really have the ovn_node container anymore...

Reply:
I asked Nadia to add this line. I need to know during upgrades whether interconnect is enabled (`--enable-interconnect`) on the ovnk node pod as well as on the ovnk master / control-plane pods. Checking ovnkube-node container is now valid only for 4.13 and 4.14 single zone, so we need to check the ovnkube-controller container in 4.14 multizone (and future versions).